SpinCore_pp/config_parser_fn.py

```diff
@@ -130,6 +130,12 @@ class configuration(object):
             None,
             "the range over which the dip lock will be performed",
         ),
+        "guessed_phalf": (
+            float,
+            "odnp_params",
+            None,
```

Comment:
what happens when this is set to None? does it freak out?

Reply:
it will freak out if it is not in the active.ini, but after setting a default it's all good

Reply:
why isn't this outdated?

Reply:
as noted below, set this to a default value